_posts/2021-10-25-g2p-applications.md

```diff
@@ -0,0 +1,79 @@
+
+Then, we convert the ReadAlong in question from its original orthographic form all the way to English IPA (well, actually to 'Arpabet' which is an ASCII-compliant phonetic transcription standard that the alignment model understands).
+
+Then, after we "do the alignment" (ie figure out which parts of the audio correspond to which part of the text), ReadAlongs puts it all back together again and ta-da! There is your aligned audio and text! Keep reading to see how to generate an X-to-English IPA mapping for your language.
+
+#### Generate your mapping between your language's IPA and English IPA
+
+After [installing]({{ "g2p-basic-mappings-local" | absolute_url }}) `g2p` and once you have made your mapping, update `g2p` by running `g2p update`. When you add or modify any mapping, `g2p` doesn't actually see it yet. `g2p update` is what scans all the mappings and makes them available to `g2p`. Then, you can run `g2p generate-mapping <input-code> --ipa`. So in our example for Kanyen'kéha (moh) we would run 
+
+```bash
+g2p generate-mapping moh --ipa
+```
+
+That will automatically generate a moh-ipa to eng-ipa mapping in `g2p/mappings/langs/generated` - do not edit this file or its configuration unless you really know what you're doing, because it will get overwritten. After generation, you can run `g2p update` again and your mapping will be [usable]({{ "g2p-basic-mappings-local" | absolute_url }}) within `g2p`!
+
+#### Visualizing the process
+
+This is actually where the introductory picture to this blog post comes from! In the graphic below, you can see the word 'bonjour' in French converted to its IPA transcription. Then, its IPA transcription gets converted into its corresponding *English* IPA transcription based on the generated mapping (notice how uvular /ʁ/ gets transformed to alveolo-palatal /ʒ/). Then finally, the English IPA form is converted to Arpabet. So in all, we have an input of 'bonjour' that gets output as 'B AO N ZH UW ZH' and the aligner runs on that form.
+
+{% picture bonjour-g2p.png %}
+
+You can recreate this type of animation using the G2P studio by selecting the French (`fra`) to English Arpabet (`eng-arpabet`) mapping, choosing 'animate' and then typing your text in the input area as seen below:
+
+{% picture g2p-animate.png %}
+
+#### What can you do with your aligned audio/text?
+
+ReadAlongs exports to a variety of formats, including epub (for e-readers), [Praat](https://www.fon.hum.uva.nl/praat/) TextGrids, [ELAN](https://archive.mpi.nl/tla/elan) files, various subtitle formats, and an embeddable web component for your website. This project is still *very* unstable though, and it is **not** yet recommended to use it in production.
```

Comment:
I guess the project is still unstable. But I feel it might be time to make the warning a little less stern. We're a lot less unstable that we were when you first wrote this!